.github/workflows/go.yml

```diff
@@ -23,7 +23,7 @@ concurrency:
 
 jobs:
     build-and-test-go-client:
-        timeout-minutes: 20
+        timeout-minutes: 35
```

Comment:
is there a reason for this?

Reply:
A previous commit in this PR hit the 20 minute limit and was cancelled: https://github.com/aws/glide-for-redis/actions/runs/8302638380/job/22725322378